frontend/src/components/user-management/student/AddClassroomModal.tsx

```diff
@@ -111,6 +111,7 @@ const AddClassroomModal = ({
             message: "New classroom created.",
             status: "success",
           });
+          window.location.reload();
```

Comment:
My strong preference is to avoid using `location.reload()`, not least because it won't show the toast the user triggered, but also it requires additional round-trips to the server. Is there a particular reason you wanted to do it this way? If the only purpose is to refresh the data in the table, I would look into using `refetchQueries` in your `useMutation` hook instead. (Have a look at the `AddAdminModal` for an example of how to do this.)